### DISCUSSION TITLE
Raspberry PI zero 2 don't boot with your script code (and on old centaur)

### DISCUSSION THREAD

**Discussion by thierryd31** [EdNekebno/DGTCentaurMods#125] [Q&A]
Hi, 
very good job!, I followed this tutorial to the letter but my raspberry pi zero 2 (not marked w) does not want to boot once the script has been passed, with the official bullseye lite image no problem, I copied the original card as requested and also it does not boot, my card nevertheless works with the original rasp, is there a difference in the bones? de boot image .. I do not understand where the error comes from and I do not understand where we configure the wifi also ?
Thanks a lot ! regards

**Comment by wormstein** [comment]
How long you waited after you have put the card in the pi?

If you used the card-setup-tool, once you put the card inside the board and boot it might take up to 15 minutes for the board to start showing the menu on the screen and to start working normally. Currently we don't have a system to tell the user that the card is self setting up.

**Reply by thierryd31** [reply]
Oh 15 min ? whaou ok i retest it thx !

**Reply by wormstein** [reply]
This will be just the first boot.

I recommend you to start all over again. Create the SD card again using the card-setup-tool.

**Comment by marcb1309** [comment]
Hi,

assuming you have followed the card-setup-tool instruction and the Centaur does not boot up after 15-20min you can power off the Centaur board disconnect the battery and remove the SDCard. Plug the SDCard into a card reader and upload the centaur_install.log from  the SDCard FAT32 boot drive. This would definetely help to figure out what went wrong.

On the other hand you wrote you do not know how to add your Wifi and Password - If you do not add your Wifi before plugging the SDCard into the Centaur the debian OS update won´t work and thus the install will fail.

In the "Raspberrry Pi Imager" Program - the RaspBerry official Imaging tool which allows you to load the bullseye lite image to the SDCard - you have to enter the hidden Menu to add WIFI name and Password.

see https://github.com/EdNekebno/DGTCentaurMods/tree/master/tools/card-setup-tool

_In addition you should add your WIFI ID and password within the hidden menu by pressing "CNTRL" + "SHIFT" + "x" at the same time._

**Reply by thierryd31** [reply]
I not read this step thx for wifi , i retest it

**Reply by wormstein** [reply]
Read the instructions inside the PowerShell window (the blue one) when you run the tool

**Comment by wormstein** [comment]
hey, and also make sure you have good wifi signal and keep the board plugged in to the power while the board is starting. I've had this kind of cases before.
Self-setup will stop in case of the above situations...

**Comment by marcb1309** [comment]
Sorry for not catching this any earlier _"(and on old centaur)"_. 

The First Centaur board came out with more visible dark rings on the playing board. Those first version Centaur boards used a display with HINK-**E09A01T1** printed on the flat cable connecting it to the DGT main board.

The second "edition" of the DGT Centaur, which has **far less visible rings on the playing field**, comes with a replacement of this "first edition" display which has now **WFT0290CZ10LW** written on the Flat cable. Due to the changed epaper also the centaur software was changed on those newer DGT Centaurs to cope with the „new“ display.

Unfortunately the DGTCentaurmods requires this second Centaur Version display. 
Therefore the only way to run the DGTCentaurMods on „first edition“ board is to replace the epaper display with exactly the same epaper as in the centaur second edition AND get hold of the original centaur software from the second edition board.

All this said - Except someone can modify the DGTCentaurMods software to also drive the "first edition" of epaper display with HINK-E09A01T1 written on the Flat connector cable.

**Reply by thierryd31** [reply]
Thanks for difference centaur, I did not know

**Reply by marcb1309** [reply]
If you should own a „first edition“ centaur board and you can live with the DGTCentaurMods menu only showing up on the web gui you should be ok without replacing the display. To show the web gui find out the IP address of your pi zero w2 on your Wi-Fi router and connect to http://<replace with IP address>:5000

**Reply by thefoster** [reply]
> The second "edition" of the DGT Centaur, which has **far less visible rings on the playing field**, comes with a replacement of this "first edition" display which has now **WFT0290CZ10LW** written on the Flat cable. Due to the changed epaper also the centaur software was changed on those newer DGT Centaurs to cope with the „new“ display.

Could you find a product that matches this display?
I'm struggeling on waveshare.com in even identify that display (SKU 16184? 12563?).

**Reply by wormstein** [reply]
iirc there are some unsers of the mod on discord that managed to find a compatible screen for the verion 1 of the board
